### ISSUE TITLE
String discrepancy between principles and techniques

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/publ-a11y#438]
Qui una lista di stringhe che sono discrepanti tra il documento dei principles e delle techniques.

- ID `conformance-certifier-credentials-prefix`:
    - in the principles I find "The certifier's credential is"
    - in the techniques "with a credential of"
- ID `navigation-structural`:
    - in the principles I find "headings"
    - in the techniques "structural navigation"
- ID `navigation-page-navigation`:
    - in the principles I find "page list."
    - in the techniques "supports page navigation"
- ID `charts-diagrams-formulas-unknown`:
    - in the principles I find "Accessibility of formulas, charts, and diagrams unknown."
    - in the techniques "The accessibility of formulas, charts, and diagrams is not available."
- ID `hazards-motion`:
    - in the principles I find "motion simulation"
    - in the techniques "motion"

**Comment by GeorgeKerscher** [comment]
Here is an update. PR soon to come:
•	ID conformance-certifier-credentials-prefix: 
o	in the principles I find "The certifier's credential is"
o	in the techniques "with a credential of"

GK: Fixed the above!

•	ID navigation-structural: 
o	in the principles I find "headings"
o	in the techniques "structural navigation"

GK: We discussed that structural navigation does not mean much to most users. We decided to change this to headings for compact. I did not change in principles.


•	ID navigation-page-navigation: 
o	in the principles I find "page list."
o	in the techniques "supports page navigation"

GK: made the above change!

•	ID charts-diagrams-formulas-unknown: 
o	in the principles I find "Accessibility of formulas, charts, and diagrams unknown."
o	in the techniques "The accessibility of formulas, charts, and diagrams is not available."

GK: I did not include the full stop in principles, suggest that this be removed in techniques.

•	ID hazards-motion: 
o	in the principles I find "motion simulation"
o	in the techniques "motion"

GK: I made the above change!

**Comment by gregoriopellegrino** [comment]
My comments:

ID `charts-diagrams-formulas-unknown`:
- in the principles I find "Accessibility of formulas, charts, and diagrams unknown."
- in the techniques "The accessibility of formulas, charts, and diagrams is not available."

GP: the two strings not only differ, but they say two different things, in one case it says they are unknown, in the other it says they are not accessible.

ID `hazards-motion`:
- in the principles I find "motion simulation"
- in the techniques "motion"

GP: maybe it is better to use motion simulation instead of just motion.

**Comment by GeorgeKerscher** [comment]
on line 45 of the spreadsheet, this is marked as false. The reason is because there is a full stop in both the ONIX and in EPUB.

I suggest that all three say:
accessibility of formulas, charts, math, and diagrams is not identified as being accessible

I will move forward with this string, unless I am told otherwise.

**Comment by clapierre** [comment]
George, did you fix this, because this morning when I resolved the conflict I removed "math" from the list unknowingly when I removed the final period.  If math was to be included please add it back in the principle's document.



Thanks

Charles

EOM



________________________________

**Comment by GeorgeKerscher** [comment]
Which file did you remove it in?/G

**Comment by clapierre** [comment]
This happened in the Pull request Small fixes for localization by gregoriopellegrino · Pull Request #446 · w3c/publ-a11y<https://github.com/w3c/publ-a11y/pull/446>

The ONIX file was modified without the math  with the id charts-diagrams-formulas-unknown




Thanks

Charles

EOM



________________________________

**Comment by GeorgeKerscher** [comment]
I think we are good now.
 
Best
George

**Comment by GeorgeKerscher** [comment]
I believe we have allstrings harmonized.
